### DISCUSSION TITLE
release v0.16 no wide band TX ?

### DISCUSSION THREAD

**Discussion by Bigpimp347** [egzumer/uv-k5-firmware-custom#87] [General]
when i first bought my UV-K5 it was to use on 4m and 6m and try on 23cms,
which under test's it does.
just installed the v0.16 and now although i have SSB but no longer have TX as before ??

am i missing something ??

how can i get wide TX back ?? 50MHz to 1.3GHz as before, anything below 50MHz was useless..

**Comment by ErikS-web** [comment]
Indeed no wide TX. For a reason. See [Issue 68](https://github.com/egzumer/uv-k5-firmware-custom/issues/68)

If you want it, it's better to try another firmware

**Reply by Bigpimp347** [reply]
i like the 23cms frequency display,
the demod is not important to me,
i tried earlier firmware which was good for TX where i needed it.
but the signal meter, modulation meter and complete 1300GHz frequency display makes the radio more atractive.
just TX from 50MHz and above is needed,
or at least the user to be able to pick and choose ??
i do not need TX 200MHz to 400MHz not 500MHz to 1000MHz
so some areas can be locked by user perhaps ??

**Comment by ErikS-web** [comment]
See Official release 0.17 under [Hidden Menu option 61](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Menu#hidden-menu)
